rest_framework_json_api/utils.py

```diff
@@ -186,6 +186,14 @@ def get_related_resource_type(relation):
         relation_model = relation.model
     elif hasattr(relation, 'get_queryset') and relation.get_queryset() is not None:
         relation_model = relation.get_queryset().model
+    elif (
+            getattr(relation, 'many', False) and
+            hasattr(relation.child, 'Meta') and
```

Comment:
Ah, you're right. I thought a vanilla serializer made no use of `Meta`, but I can see from the source that that is not correct. By all means, please keep `model`. 😊